### DISCUSSION TITLE
Is it possible to Add a base URL for WebDAV?

### DISCUSSION THREAD

**Discussion by polhaghverdian** [photoprism/photoprism#1818] [Q&A] [answered]
Hello,

Is there a way to change the BaseURL for the integrated WebDAV server? As of now the path is "/originals". I would like to change it to "photoprism/originals/". Is that possible?

For example: http://admin@192.168.2.100/photoprism/originals/

**Comment by lastzero** [comment, marked as answer]
No, not at the moment unless you change the code and recompile the application. Configuration and documentation are complex enough, we don't want to add on top of that.
